php-cs-fixer

```diff
@@ -40,9 +40,9 @@ set_error_handler(static function ($severity, $message, $file, $line) {
         exit(1);
     } elseif (
         \PHP_VERSION_ID < 70205
-        || \PHP_VERSION_ID >= 80100
+        || \PHP_VERSION_ID >= 80200
```

Comment:
we should update `.php-cs-fixer.php-highest.php:13:if (PHP_VERSION_ID <= 80001 || PHP_VERSION_ID >= 80100) {` too

Reply:
thanks, updated, also addressed some other minors, please have a look again :)